packages/core/src/metrics/MetricsService.ts

```diff
@@ -19,10 +19,22 @@ export class MetricsService {
 
   private publisher: MetricsPublisher;
 
+  private readonly configuration: ServiceConfiguration;
   private readonly defaultMetrics: Metrics[];
 
-  constructor(private readonly configuration: ServiceConfiguration) {
-    this.publisher = new NetworkMetricsPublisher(configuration.url);
+  constructor(appConfig: AeroGearConfiguration) {
+    const configuration = new ConfigurationHelper(appConfig).getConfig(MetricsService.ID);
```

Comment:
Maybe we could move that to method. It makes no sense to create object that is used instantly.

Reply:
+1, I think we can move the stuff in `ConfigurationHelper` to the `AeroGearConfiguration` class